content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -27,25 +39,35 @@ Our current process is as follows:
+  - The PM is in charge of moving issues from `Proposed` to `Todo` before the [fortnightly synchronous iteration planning](#fortnightly-iteration-planning).
+
+  - We hold a [fortnightly synchronous iteration planning](#fortnightly-iteration-planning), we take a look at the proposed issues together on the ["Next iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/4).
 
 - During an iteration, teammates **work on their assigned issues for the iteration in the order they are listed** in the ["Current iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/1) of the board. When starting work on a task, the teammate **updates its status column to "In Progress"** to communicate it to their team. This gives a good overview in the ["Current iteraton" view](https://github.com/orgs/sourcegraph/projects/200/views/1), which can also be viewed in [Kanban layout](https://github.com/orgs/sourcegraph/projects/200/views/1?layout=board), on how the iteration is tracking.
 
 - If one or more issues that were planned for an iteration are looking to **not get finished** (which includes testing) in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
 
+### Fortnightly iteration planning
+
+Every two weeks, usually on the Thursday before the next iteration begins, we hold an iteration planning. This is a short sync meeting to answer questions or align on the priority of issues planned for the next iteration. This includes: questions about priority, clarity, or edge cases of specific features for the next iteration; refining plans for the following Monday iteration start; surfacing any missing GitHub issues (for known items) ahead of the Monday kickoff so they can be made before Monday.
+
+This is primarily a review of already-selected issues with a `Todo` status, and not deciding which issues belong in `Todo`. This meeting is also an opportunity to review the status of our current iteration and examine if any issues should be carried over to the next iteration.
+
+Our goal is to begin the next iteration with a correctly-sized `Todo` list, and all issues have **assignees**, and **order**. As a rule of thumb, the sum of estimates for each individual should not exceed **FELIX TODO POINTSs** (out of a 10-work-day iteration) so we have enough buffer, time to reply to asks from other teams, to plan the next iteration's projects, and self-organized slack time.
```

Comment:
Would it be enough to document that the baseline is posted in Slack at the end of each iteration?

Reply:
Do you think the baseline will change a lot after we onboard Chris and Leo? I'm fine with slack for now but would love a quick reference when I'm planning iterations – usually spend a little while trying to find it in slack.